admin-dev/themes/new-theme/js/pages/product/components/combination-modal/CombinationModal.vue

```diff
@@ -238,15 +260,15 @@
             this.isFormUpdated = true;
           });
 
-          this.$refs.iframe.contentDocument.addEventListener('datepickerChange', () => {
+          (<HTMLIFrameElement> this.$refs.iframe)?.contentDocument?.addEventListener('datepickerChange', () => {
```

Comment:
This syntax is quite complex and hard to read, maybe we should introduce a function or getter? like:
```suggestion
          this.getIframe().contentDocument?.addEventListener('datepickerChange', () => {
```

Reply:
or maybe even:
```suggestion
          this.getIframeDocument().addEventListener('datepickerChange', () => {
```